cmd/neofs-adm/internal/modules/morph/initialize_deploy.go

```diff
@@ -188,7 +189,15 @@ func (c *initializeContext) updateContracts() error {
 
 		params := c.getAlphabetDeployItems(i, len(c.Wallets))
 		emit.Array(w.BinWriter, params...)
-		emit.Opcodes(w.BinWriter, opcode.LDSFLD1, opcode.LDSFLD0)
+
+		alphaCs.Manifest.Groups = baseGroups
```

Comment:
could you, please, clarify a little why we first do `baseGroups := alphaCs.Manifest.Groups`, do nothing with this vars and then do `alphaCs.Manifest.Groups = baseGroups`